pkg/test/case.go

```diff
@@ -397,9 +402,17 @@ func (t *Case) determineNamespace() *namespace {
 	if t.PreferredNamespace == "" {
```

Comment:
If the latter comment will be inside the else block, please move the one above into the if block for consistency.

Reply:
The comments are all outside